setup.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+##############################################################################
+# Setup script for CEPCSW:
+# - setup the external libraries
+#
+# Usage:
+# $ source setup.sh
+# or:
+# $ source setup.sh 97.0.2
```

Comment:
Again, here I would suggest to remove this option 97.0.2, since this number only matters to the internal LCG version, which is less relevant to CEPCSW it self. If you really want, we can make it as setup.sh v0.1 to connect to the upcoming release.

Reply:
Hi @xshi , the option 97.0.2 is optional. With this option, we could test the CEPCSW with different versions of LCG.

Reply:
Hi @xshi , the option is removed from the script. See commit 04d781ad8613bdb005fccb3c546243ff705b425b.